modules/security-aai/src/main/java/org/opencastproject/security/aai/ConfigurableLoginHandler.java

```diff
@@ -298,6 +326,18 @@ public void updated(Dictionary properties) throws ConfigurationException {
       roleOrganizationSuffix = CFG_ROLE_ORGANIZATION_SUFFIX_DEFAULT;
       logger.info("AAI organization membership role suffix '{}' is not configured, using default '{}'",
               CFG_ROLE_ORGANIZATION_SUFFIX_KEY, roleOrganizationSuffix);
+    }
+	
+	String cfgRoleAffiliationPrefix = StringUtils.trimToNull((String) properties.get(
```

Comment:
Indentation

Reply:
Thank you for bringing it to my attention. Fixed all the indentations :)